Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM node:20-slim AS base
+ENV PNPM_HOME="/pnpm"
+ENV PATH="$PNPM_HOME:$PATH"
+RUN corepack enable
+
+FROM base AS build
+## needed for node-gyp to build
+RUN apt-get update && \
+    apt-get install -y python3 make gcc g++
+COPY . /usr/src/app
+WORKDIR /usr/src/app
+RUN --mount=type=cache,id=pnpm,target=/pnpm/store pnpm install --frozen-lockfile
+RUN pnpm run build
+RUN pnpm deploy --filter=commonwealth --prod /prod/commonwealth
+RUN mv /usr/src/app/packages/commonwealth/build /prod/commonwealth/build
+
+FROM base AS commonwealth
+COPY --from=build /prod/commonwealth /prod/commonwealth
+WORKDIR /prod/commonwealth
+EXPOSE 8080
```

Comment:
Shouldn't this be set via `PORT` env var?

Reply:
Good catch, fixed